### DISCUSSION TITLE
Latest

### DISCUSSION THREAD

**Discussion by Schwartzblat** [Schwartzblat/WhatsAppPatcher#89] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/Schwartzblat/WhatsAppPatcher/releases/tag/1.0.16'>Latest</a>.</em>

**Comment by Draak11** [comment]
Hey, this time you didn't upload the apk, how can I download it?

**Reply by Shocoo** [reply]
Follow the installation instructions in the readme

**Comment by jojosloes** [comment]
Putting an apk online would be illegal. There are instructions on how to patch a apk that you got from another source (like apk mirror)

**Comment by AnnabelWilley** [comment]
hi. This method not working in latest Whatsapp (whatsapp update apk in 18.04.2024)

**Reply by AnnabelWilley** [reply]
Please update this  code
